### PR TITLE
Future levels

### PR DESCRIPTION
**Pull request by tttppp**

### CODE REVIEW

File: rabbit-escape-engine/src/rabbitescape/levels/development/paradise/Planel-Paradise.rel [hunk 1]
```diff
@@ -0,0 +1,25 @@
+:name=Planet Paradise
+:description=Don't lose your gravity and fall off the planet just get to the core.
```
Comment (andybalaam) [inline rabbit-escape-engine/src/rabbitescape/levels/development/paradise/Planel-Paradise.rel]: Typo in this filename

File: rabbit-escape-engine/src/rabbitescape/levels/development/unsorted/Oh-we-do-like-to-be-besides-the-treeside.rel [hunk 2]
```diff
@@ -0,0 +1,26 @@
+:name=Oh we do like to be beside the tree-side
+:description=Wait, can rabbits climb trees?
+:author_name=tttppp
```
Comment (andybalaam) [inline rabbit-escape-engine/src/rabbitescape/levels/development/unsorted/Oh-we-do-like-to-be-besides-the-treeside.rel]: Extra s in filename

**Comment by andybalaam** [comment]
As mentioned in the issue, I'd much prefer it if we could load levels from the class path like we do with real levels. This would need an ls.txt file for dir generated by the make file. If you prefer we can merge this and change that afterwards.

**Comment by tttppp** [comment]
As mentioned in the issue I think this does use the classpath, in the same way that the real levels do.  I'll fix the typos now.

**Comment by tttppp** [comment]
Fixed the typos and squashed the commits.

**Comment by andybalaam** [comment]
Yes, I was completely wrong. This is perfect.

**Comment by andybalaam** [comment]
If we want cheat mode to unlock these levels we will need them to be part of the normal menu definition, but this is good for now.

**Pull request merged**